.editorconfig

```diff
@@ -121,3 +121,11 @@ resharper_csharp_braces_for_fixed=required_for_multiline
 resharper_csharp_braces_for_ifelse=required_for_multiline
 
 resharper_csharp_accessor_owner_body=expression_body
+
+[Jenkinsfile]
```

Comment:
@kuisathaverat jfyi: we try to keep the `.editorconfig` files in sync across .NET repos within Elastic. @elastic/dotnet Are we ok with this change? Do you want to sync this to other repos? I think it's ok to have this...

Reply:
Lgtm :+1:

Reply:
I do not have a strong opinion, as you preference

Reply:
Great, then let's push it as it is now. We are ok with this. 👍